testutils/keeper/keeper.go

```diff
@@ -0,0 +1,138 @@
+	tmproto "github.com/cometbft/cometbft/proto/tendermint/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authkeeper "github.com/cosmos/cosmos-sdk/x/auth/keeper"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	distrkeeper "github.com/cosmos/cosmos-sdk/x/distribution/keeper"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	testkeeper "github.com/skip-mev/chaintestutil/keeper"
+	"github.com/stretchr/testify/require"
+
+	"github.com/skip-mev/block-sdk/lanes/base"
+	"github.com/skip-mev/block-sdk/lanes/free"
+	"github.com/skip-mev/block-sdk/lanes/mev"
+	auctionkeeper "github.com/skip-mev/block-sdk/x/auction/keeper"
+	auctiontypes "github.com/skip-mev/block-sdk/x/auction/types"
+	blocksdkkeeper "github.com/skip-mev/block-sdk/x/blocksdk/keeper"
+	blocksdktypes "github.com/skip-mev/block-sdk/x/blocksdk/types"
+)
+
+// TestKeepers holds all keepers used during keeper tests for all modules
+type TestKeepers struct {
+	testkeeper.TestKeepers
+	AuctionKeeper  auctionkeeper.Keeper
+	BlockSDKKeeper blocksdkkeeper.Keeper
+}
+
+// TestMsgServers holds all message servers used during keeper tests for all modules
+type TestMsgServers struct {
+	testkeeper.TestMsgServers
+	AuctionMsgServer auctiontypes.MsgServer
+}
+
+var additionalMaccPerms = map[string][]string{
+	auctiontypes.ModuleName: nil,
+}
+
+// NewTestSetup returns initialized instances of all the keepers and message servers of the modules
+func NewTestSetup(t testing.TB, options ...testkeeper.SetupOption) (sdk.Context, TestKeepers, TestMsgServers) {
+	options = append(options, testkeeper.WithAdditionalModuleAccounts(additionalMaccPerms))
+
+	_, tk, tms := testkeeper.NewTestSetup(t, options...)
+
+	// initialize extra keeper
+	auctionKeeper := Auction(tk.Initializer, tk.AccountKeeper, tk.BankKeeper, tk.DistrKeeper, tk.StakingKeeper)
+	blocksdkKeeper := BlockSDK(tk.Initializer)
+	require.NoError(t, tk.Initializer.LoadLatest())
+
+	// initialize msg servers
+	auctionMsgSrv := auctionkeeper.NewMsgServerImpl(auctionKeeper)
+
+	ctx := sdk.NewContext(tk.Initializer.StateStore, tmproto.Header{
+		Time:   testkeeper.ExampleTimestamp,
+		Height: testkeeper.ExampleHeight,
+	}, false, log.NewNopLogger())
+
+	err := auctionKeeper.SetParams(ctx, auctiontypes.DefaultParams())
+	require.NoError(t, err)
+
+	err = blocksdkKeeper.AddLane(ctx, blocksdktypes.Lane{
+		Id:            mev.LaneName,
+		MaxBlockSpace: math.LegacyMustNewDecFromStr("0.2"),
+		Order:         0,
+	})
+	require.NoError(t, err)
+	err = blocksdkKeeper.AddLane(ctx, blocksdktypes.Lane{
+		Id:            free.LaneName,
+		MaxBlockSpace: math.LegacyMustNewDecFromStr("0.2"),
+		Order:         1,
+	})
+	require.NoError(t, err)
+
+	err = blocksdkKeeper.AddLane(ctx, blocksdktypes.Lane{
+		Id:            base.LaneName,
+		MaxBlockSpace: math.LegacyMustNewDecFromStr("0.6"),
+		Order:         2,
+	})
```

Comment:
This stuff won't get added automatically from `populateBlockSDK` in `networksuite.go`?

Reply:
no, one of the is setting a genesis file, but here we're calling the keeper methods directly since we're not initializing from anythin